config/initializers/aws.rb

```diff
@@ -0,0 +1,9 @@
+require 'aws-sdk'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.